compile/unminified/ng-img-crop.js

```diff
@@ -3230,7 +3233,13 @@ crop.directive('imgCrop', ['$timeout', 'cropHost', 'cropPubSub', function ($time
             };
 
             if (scope.chargement == null) {
-                scope.chargement = 'Chargement';
+                var language = window.navigator.userLanguage || window.navigator.language;
+                if(language.contains('fr')) {
+                    scope.chargement = 'Chargement';
```

Comment:
I just updated this pull request with the fix. Force of habit working in controllers.